### PR TITLE
TRST-QA-3: Gap Sizes

### PR DESCRIPTION
**Pull request by akshatmittal**

### CODE REVIEW

**Comment by akshatmittal** [comment]
@tbrent I'd like you to confirm these gap sizes, I also found the gap size on StRSRP1 to be incorrect in addition to the other contracts.

**Comment by tbrent** [comment]
Interesting. Will look into. Wrong gap size on `StRSRP1` would actually matter because of `StRSRVotes`.

File: contracts/p1/RToken.sol [hunk 1]
```diff
@@ -530,5 +530,5 @@ contract RTokenP1 is ComponentP1, ERC20PermitUpgradeable, IRToken {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[42] private __gap;
+    uint256[36] private __gap;
```
Comment (tbrent) [inline contracts/p1/RToken.sol:533]: Man...why EIP712Upgradeable using 52 slots :(

**Comment by tbrent** [comment]
Trust confirmed `BasketHandler` is 28

**Review by tbrent — APPROVED** [review]

**Pull request merged**